### PR TITLE
Removes admin messages for several grenade types

### PR DESCRIPTION
**Pull request by TTerc**
[qol] I guess?

## About the PR 

Title. The grenades will still be logged, they just won't display admin alerts whenever someone primes one

List of grenades that won't display admin messages:
-cleaner
-luminol
-fog
-firefighting
-metal foam
-red oxygen
-banana/cheese sandwich/corndog/etc
-tomato thrower thing (huh, that exists)

## Why's this needed? 

Because I don't give a damn a janitor used a cleaner grenade!

### CODE REVIEW

**Comment by flappybatpal** [comment]
Do we care about red oxygen? They do have a slight flashbang style effect but are also used quite heavily.

**Comment by TTerc** [comment]
Yeah I'm fine with not having admin alerts for them

**Comment by ThatFiveGuys** [comment]
Think this effects syndicate cleaner grenade logging

**Comment by TTerc** [comment]
> Think this effects syndicate cleaner grenade logging

It doesn't, the path for fake cleaner grenades is /obj/item/chem_grenade/fcleaner

File: code/modules/chemistry/tools/grenades.dm [hunk 1]
```diff
@@ -24,6 +24,7 @@
 	stamina_cost = 0
 	stamina_crit_chance = 0
 	move_triggered = 1
+	var/is_dangerous = 1
```
Comment (pali6) [inline code/modules/chemistry/tools/grenades.dm:27]: TRUE / FALSE would be cleaner

**Review by Tarmunora — APPROVED** [review]

**Review by pali6 — APPROVED** [review]

**Pull request merged**